### DISCUSSION TITLE
Onw modal with meny modes

### DISCUSSION THREAD

**Discussion by sbc640964** [filamentphp/filament#7557] [Ideas]
I have already encountered several cases where I needed this thing:

I open a modal window of the model and want the user to see their data but also be able to update some of them, browse the relationships and also be able to update.

Division into tabs, one for display and one for updating.

In short, I think there should be an open option to decide which of the components will appear on the model, the authorization should in this case go to the developer's decision on each component, which specific authorization.

**Comment by pboivin** [comment]
I think all the tools are there already to do something like this, no?

**Reply by sbc640964** [reply]
Definately not,
For example: I cannot display infolists components in the modal form of the panel.
And in general it is not possible to display a table of relations on a modal.

**Reply by pboivin** [reply]
Fair points... for now you may need to implement a custom modal to embed a full infolist or a full table inside of a modal. This could change in the near future though, with some of the new features of Livewire 3. We'll just have to wait and see. I still think you have all the tools available in Filament to implement what you describe here.

**Reply by danharrin** [reply]
You can embed an infolist in a modal though...
